### DISCUSSION TITLE
code section dating & stack of repo 😇

### DISCUSSION THREAD

**Discussion by saner99** [community/community#128738] [Mobile] [answered]
lets do the dating in the list of repository code files as in the PC version and display the programming languages used in the repository

this feature will definitely make code research easier!

(now im going to the browser or PC version, just to view the technology stack and dates of changes to some directories in a convenient format..)

im sure, im not the only one..
who suffering from the need to switch between devices and thinking about alternative applications for code research on github using mobile app...

but.. it still doesnt exist here 🤔

**Comment by esahmed21** [comment, marked as answer]
Hi @saner99 

That's a great idea. I'll create an internal issue for our team to track it. This would for sure provide more context on navigating to a repository and it's code files.

Thank you for your valuable feedback!

**Reply by IamRezaMousavi** [reply]
I completely agree! The lack of these features forces me to constantly switch between the browser and the mobile app, which makes the user experience quite frustrating :(
I hope GitHub adds these features soon
